account_reversal_usability/models/account_move.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from setuptools.config._validate_pyproject import ValidationError
+
+from odoo import _, api, fields, models
+
+
+class MoveAlreadyReversedValidationError(ValidationError):
+    pass
+
+
+class AccountMove(models.Model):
+
+    _inherit = "account.move"
+
+    to_be_reversed = fields.Boolean(
+        compute="_compute_to_be_reversed",
```

Comment:
IMO, this field can be managed by `account.move.reversal`

```python
class AccountMoveReversal(models.TransientModel):

    _inherit = "account.move.reversal"

    def reverse_moves(self):
        self.move_ids.write({"to_be_reversed": False})
        return super().reverse_moves()

```

Reply:
As agreed, we finally go for overriding the `account.move._reverse_moves`